src/pybind/mgr/dashboard/frontend/e2e/rgw/buckets.po.ts

```diff
@@ -1,5 +1,194 @@
+        // wait for bucket to be deleted
+        browser.wait(
+          Helper.EC.not(Helper.EC.presenceOf(PageHelper.getTableCell(name))),
+          10000,
+          'Timed out waiting for bucket deletion'
+        );
+      });
+  }
+
+  invalidCreate() {
+    this.navigateTo('create');
+    expect(PageHelper.getTitleText()).toBe('Create Bucket');
+
+    const nameInputField = element(by.id('bid')); // Grabs name box field
+
+    // Gives an invalid name (too short), then waits for dashboard to determine validity
+    nameInputField.sendKeys('rq');
+    browser.wait(
+      function() {
+        // Waiting for website to decide if name is valid or not
+        return nameInputField.getAttribute('class').then(function(classValue) {
+          return classValue.indexOf('ng-pending') === -1;
+        });
+      },
+      5000,
+      'Timed out waiting for dashboard to decide bucket name validity'
+    );
+
+    // Check that name input field was marked invalid in the css
+    expect(nameInputField.getAttribute('class')).toContain('ng-invalid');
+
+    // Check that error message was printed under name input field
+    expect(element(by.cssContainingText('.form-group', 'Name')).getText()).toMatch(
+      'The value is not valid.'
+    );
+
+    // Check that dashboard detects invalid owner
+    const ownerDropDown = element(by.id('owner'));
+    browser.wait(Helper.EC.elementToBeClickable(ownerDropDown), 5000);
+
+    PageHelper.moveClick(ownerDropDown); // Clicks the Owner drop down on the Create Bucket page
+    // select some valid option. The owner drop down error message will not appear unless a valid user was selected at
+    // one point before the invalid placeholder user is selected.
+    ownerDropDown
+      .all(by.css('option'))
+      .last()
+      .click();
+
+    PageHelper.moveClick(ownerDropDown); // Clicks the Owner drop down on the Create Bucket page
+    ownerDropDown
+      .all(by.css('option'))
+      .first()
+      .click(); // select the first option, which is invalid because it is a placeholder
+
+    // Check that owner drop down field was marked invalid in the css
+    expect(element(by.id('owner')).getAttribute('class')).toContain('ng-invalid');
+
+    // Check that error message was printed under owner drop down field
+    expect(element(by.cssContainingText('.form-group', 'Owner')).getText()).toMatch(
+      'This field is required.'
+    );
+
+    // Clicks the Create Bucket button but the page doesn't move. Done by testing
+    // for the breadcrumb and heading
+    PageHelper.moveClick(element(by.cssContainingText('button', 'Create Bucket'))); // Clicks Create Bucket button
+    expect(PageHelper.getTitleText()).toBe('Create Bucket');
+    expect(PageHelper.getBreadcrumbText()).toEqual('Create');
+    element(by.cssContainingText('button', 'Cancel')).click();
+
+    // content in fields seems to subsist through tests if not cleared, so it is cleared
+    nameInputField.clear();
```

Comment:
This cleanup has to be done before
```
element(by.cssContainingText('button', 'Cancel')).click();
```
otherwise the test fails.
Protractor Screenshot reporter log & image:
```
   (Session info: headless chrome=75.0.3770.100)
  (Driver info: chromedriver=2.46.628388 (4a34a70827ac54148e092aafb70504c4ea7ae926),platform=Linux 5.1.15-300.fc30.x86_64 x86_64)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebElement.clear()
    at Array.map ()
    at BucketsPageHelper.invalidCreate (/ceph/src/pybind/mgr/dashboard/frontend/e2e/rgw/buckets.po.ts:156:20)
    at UserContext. (/ceph/src/pybind/mgr/dashboard/frontend/e2e/rgw/buckets.e2e-spec.ts:52:22)
From: Task: Run it("should test invalid inputs in create fields") in control flow
From asynchronous test: 
Error
    at Suite. (/ceph/src/pybind/mgr/dashboard/frontend/e2e/rgw/buckets.e2e-spec.ts:51:5)
    at Suite. (/ceph/src/pybind/mgr/dashboard/frontend/e2e/rgw/buckets.e2e-spec.ts:46:3)
    at Object. (/ceph/src/pybind/mgr/dashboard/frontend/e2e/rgw/buckets.e2e-spec.ts:4:1)
```
![protractor-bucket-error](https://user-images.githubusercontent.com/6780162/61113989-afe7e680-a48f-11e9-914f-dbf264d97d14.png)